src/aks-preview/HISTORY.rst

```diff
@@ -14,6 +14,11 @@ Pending
 * Deprecate the alias "-r" of parameter --source-resource-id in `az aks trustedaccess rolebinding create`
 * Refactor azure service mesh related code to meet cli style requirements.
```

Comment:
Can you please move these to `1.0.0b5`? These changes would also be released in this new version.

Reply:
Addressed